packages/now-next/@types/zeit_node-file-trace/index.d.ts

```diff
@@ -0,0 +1,59 @@
+declare module '@zeit/node-file-trace' {
```

Comment:
Why was this added? It is already defined here: https://github.com/zeit/node-file-trace/blob/master/node-file-trace.d.ts

Reply:
Because it's incomplete / not correct.

Reply:
Then please update `node-file-trace` instead of creating workarounds here